lib/modules/datasource/github-release-attachments/index.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { getDigest, getPkgReleases } from '..';
+import * as githubGraphql from '../../../util/github/graphql';
+import * as _hostRules from '../../../util/host-rules';
+import { GitHubReleaseAttachmentMocker } from './test';
+import { GithubReleaseAttachmentsDatasource } from '.';
+
+jest.mock('../../../util/host-rules');
+const hostRules: any = _hostRules;
+
+const githubApiHost = 'https://api.github.com';
+
+describe('modules/datasource/github-release-attachments/index', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
```

Comment:
```suggestion
```
not longer required

Reply:
done, also for the old `github-releases` data source